build/ReleaseNotesBuilder/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using CommandLine;
+
+namespace ReleaseNotesBuilder
+{
+    internal class Program
+    {
+        internal const string SectionPrefix = "### ";
+        internal const string SecuritySection = SectionPrefix + "Security";
+        internal const string NewFeaturesSection = SectionPrefix + "New Features";
+        internal const string FixesSection = SectionPrefix + "Fixes";
+        internal const string NoticeSection = SectionPrefix + "Notice";
+
+        static void Main(string[] args)
+        {
+            var result = Parser.Default.ParseArguments<Options>(args)
+                .WithParsed(ValidateOptions)
+                .WithNotParsed(HandleParseError);
+
+            var persistentData = LoadPersisentData(result.Value.PersistentData);
+            var checksums = LoadChecksums(result.Value.Checksums);
+            var maker = new ReleaseNotesModel(persistentData, checksums);
+            var changelog = File.ReadAllLines(result.Value.Changelog).ToList();
+            var releaseVersion = ChangelogParser.Parse(changelog, maker);
```

Comment:
Could we get a try/catch at least around the Parse call? With all the string manipulation it'd be easy for an error to sneak in somewhere down the road.

Reply:
Wrapped several methods in try-catches.